.ruby-version

```diff
@@ -1 +1 @@
-2.3.0
\ No newline at end of file
+2.6.5
```

Comment:
I believe dropping a Ruby version should be a separate PR.

Reply:
In internal development, I'd agree. But for public PRs, when what we really wanna do is to fix things up and move on, I'd love to see flexibility there. That being said, if it's a blocker, I can submit separate PRs.

Reply:
In open source, it is very important to respect the maintainer's preferences otherwise we would end up hearing conflicting opinions from different people who want to push their preferences. In this project, @ashkan18 and @dblock are the decision makers (I'm just helping them run the project) and I'll leave all preference decisions up to them.

In this specific instance, it is understandable to use the least supported version so any new API introduced to Ruby won't get creeped in. I'm not also sure how it would make it more flexible since the version constraints are still there, and people who care about compatibility and want to test against 2.3 will have to change it back. I don't understand why and how the visibility of the project would affect the versioning policy.

My preference is to not have a `.ruby-version` at all so I can more easily test the library with different Ruby versions locally. However, just changing it up to the latest version sounds like one's preference is being pushed up.